### DISCUSSION TITLE
is https://vanjs.org/ built with van?

### DISCUSSION THREAD

**Discussion by backspaces** [vanjs-org/van#33] [General]
If so, it would be good to document that it is.
If not, what are the reasons? That would be helpful, giving us insights into the problems/issues.

**Comment by artydev** [comment]
Hello,
Excuse me but what is thé purpose of your question?
Did you read the Doc, tried the examples?
If so you have certainly see what VanJS IS about...
Making static web sites is  not thé purpose of neither VueJs, React, Angular or VanJS
Regards

**Reply by backspaces** [reply]
Hi @artydev. I've did, enough to get a feel for it .. and like it.
The purpose is simply see some vanjs usage "in the wild".

**Comment by Tao-VanJS** [comment]
Hi,

Thanks for reaching out! vanjs.org is built with [Mini-Van](https://vanjs.org/minivan), a slimmed-down version of VanJS that supports server-side rendering. The code can be found here: https://github.com/vanjs-org/vanjs-org.github.io/tree/master/sitegen

It was mentioned here: https://vanjs.org/about#reliability. I can mention it in the home page as well.

**Reply by backspaces** [reply]
Thanks! And using Deno! 

I've a repo with a MVC design so the models are just for data transformations. So I can test them all in Deno easily! Way easier than my prior Puppeteer approach. 

I have to thank you for such a nice architecture, which led to a couple of hours looking at van.js https://flems.io/ and all the tooling you both use. Wow!

**Reply by Tao-VanJS** [reply]
Glad you enjoy it 😁

If you have built something interesting, you're encouraged to have your examples listed here: https://vanjs.org/demo#community-examples. VanJS community will be benefited greatly from more available examples.
